course/lesson-17-while-loops/lesson.tres

```diff
@@ -0,0 +1,237 @@
+[gd_resource type="Resource" load_steps=23 format=2]
+
+[ext_resource path="res://resources/Lesson.gd" type="Script" id=1]
+[ext_resource path="res://resources/ContentBlock.gd" type="Script" id=2]
+[ext_resource path="res://resources/QuizChoice.gd" type="Script" id=3]
+[ext_resource path="res://course/Documentation.tres" type="Resource" id=4]
+[ext_resource path="res://resources/Practice.gd" type="Script" id=5]
+
+[sub_resource type="Resource" id=1]
+script = ExtResource( 2 )
+content_id = "res://course/lesson-17-while-loops/content-DizKUdOC.tres"
+title = ""
+type = 0
+text = "Our new knowledge of vectors can help move the robot move in a grid.
+
+We can define the size of a board using a [code]Vector2[/code].
+
+Each square is a [i]cell[/i]."
```

Comment:
I think I would rather state the problem clearly and mention loops from the start. In short: we've seen how we can use functions to reuse code, now we'll see how to use loops to _repeat_ code.

And then go on to the explanation that we can use vectors to move things (note that I added info about that in l16 so l17 will reinforce the idea)